CHANGELOG.md

```diff
@@ -30,6 +30,11 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - The `getters` module contains functions and a class to interact with Verifiable
   Credentials without having to hard-code the data model in your application.
 
+### Bugfixes
+
+- The Access Grants and Access Requests status is now accepted in its abbreviated
+  form, and not only as a fully qualified IRI, as allowed by the JSON-LD context.
```

Comment:
Is it worth specifying here that we did not decide to parse access grants as JSON-LD due to technical infrastructure around JSON-LD parsing? And that we're aware JSON parsing isn't quite correct here.

Reply:
Good point, that would be more visible than the internal comment I left about this.

Reply:
4532886d927ea9cf32f69f5c58cbe8f70290d526